src/frontend/packages/cloud-foundry/src/cf-entity-generator.ts

```diff
@@ -872,10 +873,15 @@ function generateCfApplicationEntity(endpointDefinition: StratosEndpointExtensio
         getMetadata: app => ({
           guid: app.metadata.guid,
           cfGuid: app.entity.cfGuid,
+          createdAt: moment(app.metadata.created_at).format('LLL'),
           name: app.entity.name,
         }),
         getLink: metadata => `/applications/${metadata.cfGuid}/${metadata.guid}/summary`,
         getGuid: metadata => metadata.guid,
+        getLines: () => ([
+          ['Name', (meta) => meta.name],
+          ['Creation  Date', (meta) => meta.createdAt]
```

Comment:
These should be nuked before committed (double name issue, parity with space and org, etc)